src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/iceberg/IcebergSchemaEvolutionService.java

```diff
@@ -3,53 +3,98 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.snowflake.kafka.connector.internal.SnowflakeConnectionService;
 import com.snowflake.kafka.connector.internal.SnowflakeKafkaConnectorException;
-import com.snowflake.kafka.connector.internal.streaming.schemaevolution.SchemaEvolutionService;
 import com.snowflake.kafka.connector.internal.streaming.schemaevolution.SchemaEvolutionTargetItems;
-import com.snowflake.kafka.connector.internal.streaming.schemaevolution.TableSchema;
-import com.snowflake.kafka.connector.internal.streaming.schemaevolution.TableSchemaResolver;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import net.snowflake.ingest.streaming.internal.ColumnProperties;
 import org.apache.kafka.connect.sink.SinkRecord;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class IcebergSchemaEvolutionService implements SchemaEvolutionService {
+public class IcebergSchemaEvolutionService {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(IcebergSchemaEvolutionService.class);
 
   private final SnowflakeConnectionService conn;
-  private final TableSchemaResolver tableSchemaResolver;
+  private final IcebergTableSchemaResolver icebergTableSchemaResolver;
 
   public IcebergSchemaEvolutionService(SnowflakeConnectionService conn) {
     this.conn = conn;
-    this.tableSchemaResolver = new IcebergTableSchemaResolver();
+    this.icebergTableSchemaResolver = new IcebergTableSchemaResolver();
   }
 
   @VisibleForTesting
```

Comment:
Actually it is not used anymore. You can safely delete it.

Reply:
I mean the constructor below marked with `@VisibleForTesting` (not affected by your changes).